parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -355,8 +374,16 @@ async def folder_create(self, path: FsPath) -> EntryID:
             await self.local_storage.set_manifest(parent.id, new_parent)
 
         # Send events
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=child.id)
+        self._send_event(
+            CoreEvent.FS_ENTRY_UPDATED, id=parent.id, reason=FSEntryUpdatedReason.FOLDER_CREATE
+        )
+        self._send_event(
+            CoreEvent.FS_ENTRY_UPDATED,
+            id=child.id,
+            reason=FSEntryUpdatedReason.FOLDER_CREATE_ENTRY_CREATION,
+            entry_id=child.id,
```

Comment:
In fact I wouldn't even put the `entry_id`/`entry_name` fields in this event:
- `entry_id` is just the same as `id` so no need to keep it
- `entry_name` is meaningful only if we consider the parent directory (given a manifest doesn't store the file name)